### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by joshkay10**
**What's in this PR?**
I've put in bare bones stuff for A PR Template. All it will do is load the `.github/pull_request_template.md` contents into the PR comments section when a new PR is created.

**Why**
During our coding conventions chit chat the idea was suggested.

**Added feature flags**
NA

**Affected issues**
NA

**How has this been tested?**
Your testing it right now, your in the test!

**Whats Next?**
Iterate iterate iterate. likely the first one will be only kinda helpful and will need to be simplified and adjusted

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# What's in this PR?
```
Comment (gxueatlassian) [inline .github/pull_request_template.md:1]: Shall we have a # Why or Context?

Comment (joshkay10) [reply .github/pull_request_template.md:1]: I accept you offer

**Review by gxueatlassian — APPROVED** [review]
I don't have other comments for now.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+# Why 
+
+# Whats Next?
```
Comment (mboudreau) [inline .github/pull_request_template.md:5]: What's next should be the last section, how this was tested should be before that

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+# How has this been tested?
```
Comment (mboudreau) [inline .github/pull_request_template.md:7]: Probably need an "Added feature flags" and an "Affected issues" section

**Comment by mboudreau** [comment]
You're on a roll, do you think we should add an issue template as part of this PR as well?

File: .github/pull_request_template.md (hunk 3 above)
Comment (krazziekay) [inline .github/pull_request_template.md:7]: Does this section tell the reviewers how to test the changes in the PR. If so, shouldn't this be **How to test**?

Comment (joshkay10) [reply .github/pull_request_template.md]: added a helpful hint under the header

**Review by mboudreau — APPROVED** [review]
Next is to create an issue template to help out disturbed :)

**Review by Harminder84 — APPROVED** [review]

**Pull request merged**